test/PowerShellEditorServices.Test.Shared/References/ReferenceFileA.ps1

```diff
@@ -1,9 +1,9 @@
-﻿. .\ReferenceFileA.ps1
-. .\ReferenceFileB.ps1
-. .\ReferenceFileC.ps1
+﻿. ./ReferenceFileA.ps1
```

Comment:
This change shouldn't be needed and I think represents a bug where .NET Core tries to allow `\` in a filename, where PowerShell would try to normalise the path... So we should deal with this properly